src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDay = 40;
+  const rentalDays = rentalDay * days;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 5th item, you shouldn't hardcode any values in variable names